.github/workflows/main_workflow.yml

```diff
@@ -208,33 +208,13 @@ jobs:
           sudo install skaffold /usr/local/bin/
           skaffold --help
 
-          # pull node image needed for the integration tests
-          docker pull node:16.7.0
-          docker pull trufflesuite/ganache:beta
```

Comment:
I suggest we pull all images here. This speeds up tests later.

Reply:
why are tests sped up if we pull here?
If they are not found locally, they will be downloaded by `docker-py` and stored for tests reusing them, or am I missing something?